apps/web/app/providers.tsx

```diff
@@ -6,19 +6,20 @@ import { SessionProvider } from "next-auth/react";
 import CacheProvider from "react-inlinesvg/provider";
 
 import useIsBookingPage from "@lib/hooks/useIsBookingPage";
-import PlainChat from "@lib/plain/dynamicProvider";
+import PlainChat from "@lib/plain/plainChat";
 
 type ProvidersProps = {
   children: React.ReactNode;
   dehydratedState: DehydratedState;
+  isPlainChatEnabled: boolean;
 };
-export function Providers({ children, dehydratedState }: ProvidersProps) {
+export function Providers({ children, dehydratedState, isPlainChatEnabled }: ProvidersProps) {
   const isBookingPage = useIsBookingPage();
 
   return (
     <SessionProvider>
       <TrpcProvider dehydratedState={dehydratedState}>
-        {!isBookingPage ? <PlainChat /> : null}
+        {!isBookingPage && isPlainChatEnabled ? <PlainChat /> : null}
```

Comment:
This will bundle the plain chat code even for self hosters that don't require it.